src/funfuzz/js/build_options.py

```diff
@@ -332,20 +341,28 @@ def generateRandomConfigurations(parser, randomizer):  # pylint: disable=invalid
             return build_options
 
 
-def getRandomValidRepo(treeLocation):  # pylint: disable=invalid-name,missing-docstring,missing-return-doc
-    # pylint: disable=missing-return-type-doc
-    validRepos = []  # pylint: disable=invalid-name
-    for repo in ["mozilla-central", "mozilla-beta"]:
-        if os.path.isfile(sps.normExpUserPath(os.path.join(
-                treeLocation, repo, ".hg", "hgrc"))):
-            validRepos.append(repo)
+def get_random_valid_repo(tree):
+    """Given a path to Mozilla Mercurial repositories, return a randomly chosen valid one.
+
+    Args:
+        tree (str): Intended location of Mozilla Mercurial repositories
+
+    Returns:
+        Path: Location of a valid Mozilla repository
+    """
+    assert isinstance(tree, Path)  # We can remove casting Path to str after moving to Python 3.6+ completely
```

Comment:
The `Path` object in 3.5 still needs to be casted into `str` before use by other APIs (e.g. `shutil`). Into 3.6+ these APIs deal with `Path` objects properly, but because we still support 3.5, the comment is still valid.